src/vm/vm.c

```diff
@@ -1645,7 +1654,16 @@ static DictuInterpretResult run(DictuVM *vm) {
 
             char path[PATH_MAX];
             if (!resolvePath(frame->closure->function->module->path->chars, fileName->chars, path)) {
-                RUNTIME_ERROR("Could not open file \"%s\".", fileName->chars);
+                // if import is not found, try to load from the project's modules directory
+
+                memset(path, 0, PATH_MAX);
+
+                if (getcwd(path, sizeof(path)) == NULL) {
+                    RUNTIME_ERROR("Could not determine current working directiory.");
+                }
+
+                strcat(path, "/dictu_modules/");
+                strcat(path, fileName->chars);
```

Comment:
We can re-use resolvePath here

```suggestion
                if (!resolvePath("dictu_modules", fileName->chars, path)) {
                    RUNTIME_ERROR("Could not open file \"%s\".", fileName->chars);
                }
```